src/main/java/omero/gateway/Gateway.java

```diff
@@ -366,69 +366,6 @@ public void disconnect() {
             cacheService.shutDown();
     }
 
-    /**
```

Comment:
insight has a method ``joinSession``. It does not seem to be used but some changes will be required there too

Reply:
I forgot about that